client/sites-dashboard/utils.ts

```diff
@@ -45,7 +45,7 @@ export const isP2Site = ( site: SiteExcerptNetworkData ) => {
 };
 
 export const isStagingSite = ( site: SiteExcerptNetworkData ) => {
```

Comment:
@wojtekn , should we update the type of the parameters?

```suggestion
export const isStagingSite = ( site: SiteExcerptNetworkData | undefined) => {
```